src/afl-fuzz-bitmap.c

```diff
@@ -484,6 +507,21 @@ u8 __attribute__((hot)) save_if_interesting(afl_state_t *afl, void *mem,
      need_hash = 1;
   s32 fd;
   u64 cksum = 0;
+  u32 cksum_simplified = 0, cksum_unique = 0;
+  u8 san_fault = 0;
+  u8 san_idx = 0;
+  u8 feed_san = 0;
+
+  afl->san_case_status = 0;
+
+  /* Mask out var bytes */
+  if (unlikely(afl->san_binary_length)) {
+    for (u32 i = 0; i < afl->fsrv.map_size; i++) {
```

Comment:
that looks very expensive for every safe_if_interesting check. rather change the function that detects varbytes and if there are then do this there.

Reply:
Yeah exactly, this is the worst hack I did due to deadline approaching. If it's the correct place to mask var bytes, I will change it to detect if any var bytes firstly.

Reply:
I would just do it here: https://github.com/AFLplusplus/AFLplusplus/blob/635cd7374e8484afbca3753d0e157287e334ab0d/src/afl-fuzz-run.c#L584

variable bytes are rare

Reply:
SAND needs stable "execution pattern" for every execution and that's why I put it here instead of calibration.

Since this is too expensive, I simply remove these makeups so it's users' responsibilities to avoid unstable targets. I will document it somewhere.